.travis.yml

```
@@ -20,12 +20,16 @@ matrix:
       - cargo clippy --examples -- -D warnings
       - cargo doc --no-deps
       - make -C examples
+      - RUSTFLAGS="-D warnings" cargo build --release --verbose
```

I think we need to use "cargo test --dry-run" here, otherwise the tests don't actually get built.

```suggestion
      - RUSTFLAGS="-D warnings" cargo test --dry-run --verbose
```